tools/openmrs_run

```diff
@@ -12,17 +12,51 @@
 
 # Starts a local OpenMRS server for development.
 
+## Usage:
+##   tools/openmrs_run [-p PORTNUM] [--debug]
+##
+## Note that it's possible to change the OpenMRS server ID by setting OPENMRS_SERVER.
+
 set -e
-SERVER_ID=server
+
+PORT="9000"
+DEBUG=0
+
+while [[ $# > 0 ]]
```

Comment:
Loops until there are no unprocessed command-line arguments. `$#` is the number of command line arguments. Every time we call `shift` the first argument is removed from the set, so `$#` is decremented.